modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaCommerceOrderConstantsCheck.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.petra.string.StringBundler;
+import com.liferay.petra.string.StringUtil;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Lázaro Costa
+ */
+public class UpgradeJavaCommerceOrderConstantsCheck
+	extends BaseUpgradeMatcherReplacementCheck {
+
+	@Override
+	protected String formatMatcherIteration(
+		String content, String newContent, Matcher matcher) {
+
+		String constant = matcher.group(1);
+		String constantCall = matcher.group();
+		String newConstantCall = "CommerceOrderPaymentConstants.";
+
+		if (constant.equals("_STATUS_AUTHORIZED")) {
+			newConstantCall = newConstantCall + "STATUS_AUTHORIZED";
+		}
+		else if (constant.equals("_STATUS_PAID")) {
+			newConstantCall = newConstantCall + "STATUS_COMPLETED";
+		}
+		else if (constant.equals("_STATUS_PENDING")) {
+			newConstantCall = newConstantCall + "STATUS_PENDING";
+		}
```

Comment:
@lazhcsilva, if we add the `_` before the parentheses on the regex, we don't need to repeat at the beginning of each if statement.

`CommerceOrderConstants.PAYMENT_([A-Z_]+)`